packages/@coorpacademy-progression-engine/src/types.js

```diff
@@ -272,6 +272,7 @@ export type QCMGraphicQuestion = $Exact<{|
 |}>;
 
 export type SliderQuestion = {|
+  ...QuestionCommon,
```

Comment:
It could be kind to have this update in `BasicQuestion` too

Reply:
Done